dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties

```diff
@@ -4,3 +4,4 @@ io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnam
 io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnamedMethodInjectionBean.SIMPLE_KEY6=VALUE6
 noConverterKey=value
 org.jboss.logging.provider=slf4j
+value1DefinedInBootstrapProperties=value1
```

Comment:
Can you update this name to something like the following and also define this in microprofile-config.properties to test that the override capabilities? Sorry, I forgot to mention last week.
`value1DefinedInTwoPlaces=value1`